qiskit/algorithms/gradients/__init__.py

```diff
@@ -17,74 +17,94 @@
 
 .. currentmodule:: qiskit.algorithms.gradients
 
-
-Estimator Gradients
-===================
+Base Classes
+============
 
 .. autosummary::
    :toctree: ../stubs/
 
    BaseEstimatorGradient
-   DerivativeType
-   FiniteDiffEstimatorGradient
-   LinCombEstimatorGradient
-   ParamShiftEstimatorGradient
-   SPSAEstimatorGradient
-   ReverseEstimatorGradient
+   BaseQGT
+   BaseSamplerGradient
+   EstimatorGradientResult
+   SamplerGradientResult
+   QGTResult
 
-Sampler Gradients
-=================
+Finite Differences
+==================
 
 .. autosummary::
    :toctree: ../stubs/
 
-   BaseSamplerGradient
+   FiniteDiffEstimatorGradient
    FiniteDiffSamplerGradient
+
+Linear Combinations
+===================
```

Comment:
Could we call these linear combination of unitaries, which is the full name of the method?
```suggestion
Linear Combination of Unitaries
===============================
```